src/datagrid/datagrid.component.scss

```diff
@@ -0,0 +1,217 @@
+@import '../../styles/colors';
```

Comment:
Should this file be split so that e.g. tooltip component would have it's own scss.